apps/accounts/templates/base-email.html

```diff
@@ -0,0 +1,140 @@
+<!doctype html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+    <head>
+        <!-- NAME: EDUCATE -->
+        <!--[if gte mso 15]>
+        <xml>
+            <o:OfficeDocumentSettings>
+            <o:AllowPNG/>
+            <o:PixelsPerInch>96</o:PixelsPerInch>
+            </o:OfficeDocumentSettings>
+        </xml>
+        <![endif]-->
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>{% block title %}Title here?{% endblock %}</title>
+
+	{% block style %}
+	    {% include "email-common-style-template.html" %}
+	{% endblock %}
+
+    </head>
+    <body>
+
+	{% block body-head %}
+    	    {% include "email-common-body-heading-template.html" %}
+	{% endblock %}
+
+                <table align="left" border="0" cellpadding="0" cellspacing="0" style="max-width:100%; min-width:100%;" width="100%" class="mcnTextContentContainer">
+                    <tbody><tr>
+                        
+                        <td valign="top" class="mcnTextContent" style="padding: 0px 18px 9px; font-style: normal; font-weight: normal;">
+                        
+                            <h2><strong id="docs-internal-guid-cc422a4d-7fff-1765-ef6f-b458cb279173">You’ve Registered Your First Application!</strong></h2>
+
+<p style="font-size: 18px !important; font-style: normal; font-weight: normal;"><strong id="docs-internal-guid-6a5a05a2-7fff-5f51-3ab2-aa3d10f3c881">Way to go, {{ FIRST_NAME }}! </strong>You’ve registered your first application with Blue Button 2.0. The next step is to make an API call to start interacting with our synthetic beneficiary data. You’ll be able to craft your app with confidence by being able to see real examples of exactly what your users will see.<br>
```

Comment:
I think you mean to specify a block for `content` here?

Reply:
TY. Updating.